### ISSUE TITLE
[freebsd-vg-autotest] build time out during Vagrant setup

### ISSUE THREAD

**Issue by optimistyzy** [spdk/spdk#1424]
## Link to the failed CI build

https://ci.spdk.io/results/autotest-per-patch/builds/16066/archive/freebsd-vg-autotest/build.log

## Execution failed at

00:00:15.750  remote: Total 4 (delta 3), reused 4 (delta 3)        
00:00:15.768  From https://review.spdk.io/spdk/spdk
00:00:15.768   * [new ref]             refs/changes/14/2714/4 -> refs/test
00:00:15.874  Note: checking out '4b090a75cac8798315da35e42cab2fa6833f3380'.
00:00:15.874  
00:00:15.874  You are in 'detached HEAD' state. You can look around, make experimental
00:00:15.874  changes and commit them, and you can discard any commits you make in this
00:00:15.874  state without impacting any branches by performing another checkout.
00:00:15.874  
00:00:15.874  If you want to create a new branch to retain commits you create, you may
00:00:15.874  do so (now or later) by using -b with the checkout command again. Example:
00:00:15.874  
00:00:15.874    git checkout -b <new-branch-name>
00:00:15.874  
00:00:15.874  HEAD is now at 4b090a75c nvmf: Let users configure max_ioqpairs_per_ctrlr instead of max_qpairs_per_ctrlr
00:00:15.901  4b090a75c nvmf: Let users configure max_ioqpairs_per_ctrlr instead of max_qpairs_per_ctrlr
00:00:15.901  477761a27 vm_setup: Drop install_iscsi_adm
00:00:15.901  6f97efb78 test/make: add leaf type changes check to check_so_deps.
00:00:15.901  4bca12f1f lib and module: remove remaining _spdk prefixes.
00:00:15.902  57c2b0c5a lib/util: remove _spdk prefix from functions.
00:00:15.936  Submodule 'dpdk' (https://github.com/spdk/dpdk.git) registered for path 'dpdk'
00:00:15.945  Cloning into '/var/jenkins/workspace/freebsd-vg-autotest/spdk/dpdk'...
00:00:19.852  Submodule path 'dpdk': checked out '76f9669b1f4cdd52bae2c3408ad129272bf99d1f'
00:00:19.892  Submodule 'intel-ipsec-mb' (https://github.com/spdk/intel-ipsec-mb.git) registered for path 'intel-ipsec-mb'
00:00:19.899  Cloning into '/var/jenkins/workspace/freebsd-vg-autotest/spdk/intel-ipsec-mb'...
00:00:20.751  Submodule path 'intel-ipsec-mb': checked out '489ec6082a9d4a65d7569d1772dce64d2e96f5b5'
00:00:20.792  Submodule 'isa-l' (https://github.com/spdk/isa-l.git) registered for path 'isa-l'
00:00:20.801  Cloning into '/var/jenkins/workspace/freebsd-vg-autotest/spdk/isa-l'...
00:00:21.641  Submodule path 'isa-l': checked out 'f3993f5c0b69118a229c2dfbb360515cd34a24e6'
00:00:21.680  Submodule 'ocf' (https://github.com/Open-CAS/ocf.git) registered for path 'ocf'
00:00:21.688  Cloning into '/var/jenkins/workspace/freebsd-vg-autotest/spdk/ocf'...
00:00:23.750  Submodule path 'ocf': checked out '9d0795564082b5dfb489ab3fc2fa22cb1538ab85'
00:00:23.767  [freebsd-vg-autotest] $ /usr/bin/env bash /tmp/jenkins1464839425429154739.sh
00:00:23.771  + cat
00:00:23.788  [freebsd-vg-autotest] $ /usr/bin/env bash /tmp/jenkins7657143463590074362.sh
00:00:23.792  + [[ -n '' ]]
00:00:23.811  [freebsd-vg-autotest] $ /usr/bin/env bash /tmp/jenkins4866079484300993653.sh
00:00:23.815  + [[ -f /var/jenkins/workspace/freebsd-vg-autotest/spdk/test/blobfs/rocksdb/rocksdb_commit_id ]]
00:00:23.815  ++ cat /var/jenkins/workspace/freebsd-vg-autotest/spdk/test/blobfs/rocksdb/rocksdb_commit_id
00:00:23.818  + rocks_hash=526c73bd94150cc8fbd651f736e1ca95f50d8e13
00:00:23.818  + grep SPDK_TEST_BLOBFS=1 /var/jenkins/workspace/freebsd-vg-autotest/autorun-spdk.conf
00:00:23.834  [freebsd-vg-autotest] $ /usr/bin/env bash /tmp/jenkins8635034008144122611.sh
00:00:23.856  [freebsd-vg-autotest] $ /usr/bin/env bash /tmp/jenkins5416038633897479829.sh
00:00:23.860  + backend_dir=/var/lib/libvirt/images/backends
00:00:23.860  + ex_num=7
00:00:23.860  + cd /var/jenkins/workspace/freebsd-vg-autotest/spdk
00:00:23.860  + git submodule update --init
00:00:23.942  + git log --oneline -n5
00:00:23.945  4b090a75c nvmf: Let users configure max_ioqpairs_per_ctrlr instead of max_qpairs_per_ctrlr
00:00:23.945  477761a27 vm_setup: Drop install_iscsi_adm
00:00:23.945  6f97efb78 test/make: add leaf type changes check to check_so_deps.
00:00:23.945  4bca12f1f lib and module: remove remaining _spdk prefixes.
00:00:23.946  57c2b0c5a lib/util: remove _spdk prefix from functions.
00:00:23.946  ++ git log --oneline --decorate=full
00:00:23.947  ++ grep -oP '(?<=refs/remotes/)origin/v.+(?=\))'
00:00:23.947  ++ head -1
00:00:24.165  + spdk_branch=
00:00:24.165  + [[ '' < origin/v20.04.x ]]
00:00:24.165  + cp -r /home/sys_sgci/spdk-20-04-ref/scripts/vagrant /var/jenkins/workspace/freebsd-vg-autotest/spdk/scripts
00:00:24.169  + trap 'cd freebsd11-libvirt; vagrant destroy -f' SIGTERM SIGINT EXIT
00:00:24.169  + cd /var/jenkins/workspace/freebsd-vg-autotest
00:00:24.170  + vagrant_start='./spdk/scripts/vagrant/create_vbox.sh -n 10 -s 12288 -x http://proxy-chain.intel.com:911 -p libvirt   --qemu-emulator=/usr/local/qemu/spdk-5.0.0/bin/qemu-system-x86_64   -b /var/lib/libvirt/images/backends/ex7-nvme.img,nvme,1'
00:00:24.170  + [[ freebsd11 =~ fedora ]]
00:00:24.170  + vagrant_start+=' -H -a -u -v freebsd11'
00:00:24.170  + ./spdk/scripts/vagrant/create_vbox.sh -n 10 -s 12288 -x http://proxy-chain.intel.com:911 -p libvirt --qemu-emulator=/usr/local/qemu/spdk-5.0.0/bin/qemu-system-x86_64 -b /var/lib/libvirt/images/backends/ex7-nvme.img,nvme,1 -H -a -u -v freebsd11
00:00:24.205  mkdir: created directory '/var/jenkins/workspace/freebsd-vg-autotest/freebsd11-libvirt'
00:00:24.205  /var/jenkins/workspace/freebsd-vg-autotest/freebsd11-libvirt /var/jenkins/workspace/freebsd-vg-autotest
00:00:25.532  vagrant-proxyconf already installed... skipping
00:00:28.186  Bringing machine 'default' up with 'libvirt' provider...
00:00:31.448  ==> default: Creating image (snapshot of base box volume).
00:00:31.609  ==> default: Creating domain with the following settings...
00:00:31.610  ==> default:  -- Name:              freebsd11-libvirt_default_1591070914_28fc4ea0b98d14d6048c
00:00:31.610  ==> default:  -- Domain type:       kvm
00:00:31.611  ==> default:  -- Cpus:              10
00:00:31.611  ==> default:  -- Feature:           acpi
00:00:31.611  ==> default:  -- Feature:           apic
00:00:31.611  ==> default:  -- Feature:           pae
00:00:31.611  ==> default:  -- Memory:            12288M
00:00:31.612  ==> default:  -- Memory Backing:    hugepages: 
00:00:31.612  ==> default:  -- Management MAC:    
00:00:31.612  ==> default:  -- Loader:            
00:00:31.612  ==> default:  -- Nvram:             
00:00:31.613  ==> default:  -- Base box:          spdk/freebsd11
00:00:31.613  ==> default:  -- Storage pool:      default
00:00:31.613  ==> default:  -- Image:             /var/lib/libvirt/images/freebsd11-libvirt_default_1591070914_28fc4ea0b98d14d6048c.img (32G)
00:00:31.614  ==> default:  -- Volume Cache:      default
00:00:31.614  ==> default:  -- Kernel:            
00:00:31.614  ==> default:  -- Initrd:            
00:00:31.615  ==> default:  -- Graphics Type:     vnc
00:00:31.615  ==> default:  -- Graphics Port:     -1
00:00:31.615  ==> default:  -- Graphics IP:       127.0.0.1
00:00:31.615  ==> default:  -- Graphics Password: Not defined
00:00:31.616  ==> default:  -- Video Type:        cirrus
00:00:31.616  ==> default:  -- Video VRAM:        9216
00:00:31.616  ==> default:  -- Sound Type:	
00:00:31.616  ==> default:  -- Keymap:            en-us
00:00:31.617  ==> default:  -- TPM Path:          
00:00:31.617  ==> default:  -- INPUT:             type=mouse, bus=ps2
00:00:31.617  ==> default:  -- Command line args: 
00:00:31.618  ==> default:     -> value=-drive, 
00:00:31.618  ==> default:     -> value=format=raw,file=/var/lib/libvirt/images/backends/ex7-nvme.img,if=none,id=nvme-0, 
00:00:31.619  ==> default:     -> value=-device, 
00:00:31.619  ==> default:     -> value=nvme,drive=nvme-0,serial=12340, 
00:00:34.634  ==> default: Creating shared folders metadata...
00:00:34.662  ==> default: Starting domain.
00:00:36.113  ==> default: Waiting for domain to get an IP address...
00:01:00.456  ==> default: Waiting for SSH to become available...
00:01:09.781      default: 
00:01:09.781      default: Inserting generated public key within guest...
00:01:10.770      default: Removing insecure key from the guest if it's present...
00:01:11.855      default: Key inserted! Disconnecting and reconnecting using new SSH key...
00:01:12.223  ==> default: Configuring and enabling network interfaces...
00:01:18.257  ==> default: Exporting NFS shared folders...
00:01:18.260  ==> default: Preparing to edit /etc/exports. Administrator privileges will be required...
00:01:19.026  ==> default: Mounting NFS shared folders...
00:15:00.399  Build timed out (after 15 minutes). Marking the build as failed.
00:15:00.407  Build was aborted
00:15:00.408  [PostBuildScript] - [INFO] Executing post build scripts.
00:15:00.415  [freebsd-vg-autotest] $ /usr/bin/env bash /tmp/jenkins3849125787360553341.sh
00:15:00.418  + mv /var/jenkins/workspace/freebsd-vg-autotest/freebsd11-libvirt/output /var/jenkins/workspace/freebsd-vg-autotest/output
00:15:00.439  [freebsd-vg-autotest] $ /usr/bin/env bash /tmp/jenkins8080847223591207687.sh
00:15:00.501  + cd freebsd11-libvirt
00:15:00.501  + vagrant destroy -f

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_16066.html

**Comment by tomzawadzki** [comment]
[Bug scrub] This failure is related to Vagrant running the VMs.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_16736.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_16974.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_18637.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @peluse. log: https://ci.spdk.io/public_build/autotest-per-patch_18628.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_19179.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jkalwas. log: https://ci.spdk.io/public_build/autotest-per-patch_19538.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @peluse. log: https://ci.spdk.io/public_build/autotest-per-patch_19527.html

**Comment by tomzawadzki** [comment]
> Another instance of this failure. Reported by @peluse. log: https://ci.spdk.io/public_build/autotest-per-patch_19527.html

This log is unrelated, it does not pertain to Vagrant at all. Tho there is another interesting failure there:
```
00:00:02.336  warning: redirecting to https://review.spdk.io/gerrit/spdk/spdk/
00:11:19.353  git version 2.21.1
```

> Another instance of this failure. Reported by @peluse. log: https://ci.spdk.io/public_build/autotest-per-patch_18628.html

This log is not related to Vagrant as well.

@karlatec All other logs point to Vagrant setup for FreeBSD job:
```
00:01:25.032  ==> default: Mounting NFS shared folders...
00:15:00.313  Build timed out (after 15 minutes). Marking the build as failed.
```
Can you (or @wawryk @mikeBashStuff) please see for anything specific for this VM or if there is possibility for more information what went wrong ?

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @wawryk. log: https://ci.spdk.io/public_build/autotest-per-patch_20180.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jkalwas. log: https://ci.spdk.io/public_build/autotest-per-patch_20987.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_21340.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_21386.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jkalwas. log: https://ci.spdk.io/public_build/autotest-per-patch_21406.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_21414.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_21438.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_21959.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @yidong0635. log: https://ci.spdk.io/public_build/autotest-per-patch_22000.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @Donaim. log: https://ci.spdk.io/public_build/autotest-per-patch_22027.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @Donaim. log: https://ci.spdk.io/public_build/autotest-per-patch_21989.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @yidong0635. log: https://ci.spdk.io/public_build/autotest-per-patch_22171.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_22308.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_23105.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @wawryk. log: https://ci.spdk.io/public_build/autotest-per-patch_23211.html

**Comment by karlatec** [comment]
Same issue is bugging nightly builds, but with much greater frequency so I finally gathered my strength to take a look at it.

All VM hosts (for per-patch and nightly VMs) use the same system - Ubuntu 18.04 LTS.
Turns out that mounting NFS shares got stuck because of the firewall. Which is weird, because on all systems UFW service was in `"active/exited"` state and `ufw status verbose` reported `Status: inactive`. For some reason NFS still got stuck on firewall, even though it reported to be inactive.

Long story short: `service ufw stop` fixed this issue for nightly test VMs. I did the same for per-patch VM hosts as well, let's see if that helps.

**Comment by tomzawadzki** [comment]
[Bug scrub] Firewall was disabled for NFS shares, patches should no longer encounter this issue. Observing is further reports occur.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_23258.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @glodfish-zg. log: https://ci.spdk.io/public_build/autotest-per-patch_23260.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @glodfish-zg. log: https://ci.spdk.io/public_build/autotest-per-patch_23263.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_23265.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @Seth5141. log: https://ci.spdk.io/public_build/autotest-per-patch_23763.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @Seth5141. log: https://ci.spdk.io/public_build/autotest-per-patch_23782.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @Seth5141. log: https://ci.spdk.io/public_build/autotest-per-patch_23820.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_23926.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @yidong0635. log: https://ci.spdk.io/public_build/autotest-per-patch_24053.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_24014.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_24134.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_24173.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_24165.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @Seth5141. log: https://ci.spdk.io/public_build/autotest-per-patch_24218.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @yidong0635. log: https://ci.spdk.io/public_build/autotest-per-patch_24291.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_24294.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_24314.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_24385.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_24475.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @karlatec. log: https://ci.spdk.io/public_build/autotest-per-patch_24494.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @karlatec. log: https://ci.spdk.io/public_build/autotest-per-patch_23964.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @Donaim. log: https://ci.spdk.io/public_build/autotest-per-patch_24577.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_24749.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @Seth5141. log: https://ci.spdk.io/public_build/autotest-per-patch_24837.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_24959.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_25011.html

**Comment by yidong0635** [comment]
Does this can give any helps? same issue with freebsd vagrant file mounting :
https://github.com/wunki/vagrant-freebsd/issues/4

**Comment by karlatec** [comment]
@yidong0635 some of this fixes no longer apply. I tried some of them before with no success

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_25317.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_25324.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_25429.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @optimistyzy. log: https://ci.spdk.io/public_build/autotest-per-patch_25497.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_25648.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_25659.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_25698.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_25700.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @wawryk. log: https://ci.spdk.io/public_build/autotest-per-patch_25851.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jimharris. log: https://ci.spdk.io/public_build/autotest-per-patch_25862.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jimharris. log: https://ci.spdk.io/public_build/autotest-per-patch_25864.html

**Comment by jimharris** [comment]
@karlatec - any help needed to get to the bottom of this one?  It sounds like you got this fixed for nightly tests - can you confirm its still working there?

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @wawryk. log: https://ci.spdk.io/public_build/autotest-per-patch_25854.html

**Comment by karlatec** [comment]
@jimharris Yeah, I can't really figure this one out. Similar issues with FreeBsd Vagrant boxes reported I've found on the Internet suggest that most commonly this is caused by firewall settings on the host system. I already tried both setting proper firewall entries and disabling it completely too, but with no results.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_25954.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_25986.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_26451.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_26463.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jimharris. log: https://ci.spdk.io/public_build/autotest-per-patch_26539.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jkalwas. log: https://ci.spdk.io/public_build/autotest-per-patch_26606.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_26633.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_26632.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jimharris. log: https://ci.spdk.io/public_build/autotest-per-patch_26658.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jkalwas. log: https://ci.spdk.io/public_build/autotest-per-patch_26678.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_26691.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_26696.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @yidong0635. log: https://ci.spdk.io/public_build/autotest-per-patch_26717.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @benlwalker. log: https://ci.spdk.io/public_build/autotest-per-patch_26778.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_26790.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_26885.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @floatious. log: https://ci.spdk.io/public_build/autotest-per-patch_26872.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_26892.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jimharris. log: https://ci.spdk.io/public_build/autotest-per-patch_27132.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_26999.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://ci.spdk.io/public_build/autotest-per-patch_27351.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @benlwalker. log: https://ci.spdk.io/public_build/autotest-per-patch_27276.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @AlekseyMarchuk. log: https://ci.spdk.io/public_build/autotest-per-patch_27497.html

**Comment by karlatec** [comment]
Made a "slight" change on CI configuration side. So far not a single Vagrant/NFS timeout for the whole day. Let's see for a bit longer if this keeps up.

**Comment by karlatec** [comment]
No reports in 2 days. @mikeBashStuff looks like our "small" change helped out a lot

**Comment by mikeBashStuff** [comment]
@karlatec Cool! \o/ let's hope it won't break in the long run. :)

**Comment by tomzawadzki** [comment]
Thanks !
